composer.json

```diff
@@ -18,22 +18,11 @@
         "ext-simplexml": "*",
         "guzzle/guzzle": "^2.8|^3.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.0|^3.0",
+        "symfony/config": "^2.6|^3.0",
```

Comment:
> symfony/config 2.6 supports PHP 5.3.3. What's the problem?

You must understand how deps works. It is not that "Oh, it works on 2.6! I will require it even if I don't depend on features of 2.6!" Be aware that sb might have in his deps both symfony/config and php-coveralls, and then bumping the version of dep when not needed is not cool.

Due to given argument in second comment - :+1: for 2.4